flink-examples/flink-examples-streaming/src/main/java/org/apache/flink/streaming/examples/iteration/IterateExample.java

```diff
@@ -96,7 +100,21 @@ public static void main(String[] args) throws Exception {
         } else {
             System.out.println("Executing Iterate example with default input data set.");
             System.out.println("Use --input to specify file input.");
-            inputStream = env.addSource(new RandomFibonacciSource());
+
+            GeneratorFunction<Long, Tuple2<Integer, Integer>> dataGenerator =
+                    new RandomFibonacciGenerator();
+            DataGeneratorSource<Tuple2<Integer, Integer>> generatorSource =
+                    new DataGeneratorSource<>(
+                            dataGenerator,
+                            BOUND,
+                            RateLimiterStrategy.perSecond(20),
+                            TypeInformation.of(new TypeHint<Tuple2<Integer, Integer>>() {}));
```

Comment:
Could `Types.TUPLE(Types.INT, Types.INT)` work here? If possible, it would be more concise.